Brink Innovation/Example CHT application/disease-surveillance/test/constants.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Nice, maybe you can make use of this in the whole project (not only for tests) ?
What about having constant containing all forms too?